resources/views/list-view/list-view.blade.php

```diff
@@ -1,7 +1,7 @@
 <x-lv-layout>
   {{-- Search input and filters --}}
-  <div class="px-4">
-    @include('laravel-views::table-view.filters')
+  <div class="pl-4">
```

Comment:
The padding on both sides was to align it up with the list items, do you think looks better this way?